exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Column.kt

```diff
@@ -181,7 +181,6 @@ class Column<T>(
 
         if (table != other.table) return false
         if (name != other.name) return false
-        if (columnType != other.columnType) return false
```

Comment:
Like we discussed, this is the only line I'm hesitant about, but I agree that it makes sense for what should be.
It'll be interesting to see if we run into any edge cases affected by its removal.

Reply:
I'll be waiting for issues =)